requirements/linux.yml

```diff
@@ -0,0 +1,5 @@
+# Linux specific dependencies
+channels:
+  - conda-forge
+dependencies:
+  - pyxdg >=0.26
```

Comment:
Please remove this file and instead add the last line to  `requirements/posix.txt`.

Reply:
Wouldn't that affect macos as well?